### PR TITLE
Add sponsor images

### PR DESCRIPTION
**Pull request by schmijos**
I now simply load the sponsor logos from the public folder `public/images`. `logo_source` can be a URLs as well, if you want.

### CODE REVIEW

File: app/views/mobile_app/sensors/sponsor.json.jbuilder [hunk 1]
```diff
@@ -1 +1,2 @@
 json.extract! @sponsor, :id, :name, :description, :created_at
+json.logo_url image_url(@sponsor.logo_source || 'coredump.png')
```
Comment (dbrgn) [inline app/views/mobile_app/sensors/sponsor.json.jbuilder:2]: Does this mean that the logo will always be set to `coredump.png` if unset? In that case, it should probably be `null` instead.

File: app/views/sponsors/show.json.jbuilder [hunk 2]
```diff
@@ -1 +1,2 @@
 json.extract! @sponsor, :id, :name, :description, :active, :sensor_ids, :created_at, :updated_at
+json.logo_url image_url(@sponsor.logo_source || 'coredump.png')
```
Comment (dbrgn) [inline app/views/sponsors/show.json.jbuilder:2]: Does this mean that the logo will always be set to `coredump.png` if unset? In that case, it should probably be `null` instead.

File: test/controllers/mobile_app/sensors_controller_test.rb [hunk 3]
```diff
@@ -129,12 +129,13 @@ class SensorsControllerTest < ActionDispatch::IntegrationTest
     end
 
     test 'should show active sponsor of a sensor' do
-      create(:sponsor, active: true, sensors: [@sensor])
+      create(:sponsor, active: true, logo_source: 'hsr.svg', sensors: [@sensor])
```
Comment (dbrgn) [inline test/controllers/mobile_app/sensors_controller_test.rb:132]: Just as a sidenote: The SVG is fine for testing, but at least on Android, rendering SVGs without using a WebView is not possible without including a large library. (SVG is quite a complex standard.) Therefore for production we probably have to stick to PNGs :slightly_frowning_face:

**Review by dbrgn — COMMENTED** [review]
Perfect, I like the approach of supporting both a local file and a URL :slightly_smiling_face:

File: app/views/mobile_app/sensors/sponsor.json.jbuilder (hunk 1 above)
Comment (schmijos) [reply app/views/mobile_app/sensors/sponsor.json.jbuilder:2]: I more thought of a fallback avatar. Looks nicer and makes the handling in the client easier.

Comment (dbrgn) [reply app/views/mobile_app/sensors/sponsor.json.jbuilder:2]: Yeah, but it doesn't really make sense to write "Hey, this sponsor is sponsored by HSR" and then put our own logo next to it :slightly_smiling_face: I'd return `null` instead, then the client can decide what to show.

Comment (schmijos) [reply app/views/mobile_app/sensors/sponsor.json.jbuilder]: See https://github.com/gfroerli/api/pull/70/commits/21725b23a30f37e555c2b921d8c0afb813a946c8

**Review by dbrgn — APPROVED** [review]

**Pull request merged**